providers/vic/proxy/isolation_proxy.go

```diff
@@ -31,13 +31,16 @@ import (
 
 	"github.com/docker/docker/api/types/strslice"
 
+	"time"
+
 	"github.com/virtual-kubelet/virtual-kubelet/providers/vic/cache"
 	"github.com/virtual-kubelet/virtual-kubelet/providers/vic/constants"
 	"github.com/vmware/vic/pkg/vsphere/sys"
 )
 
 type IsolationProxy interface {
 	CreateHandle(op trace.Operation) (string, string, error)
+	Remove(op trace.Operation, containerID string, force bool) error
```

Comment:
Can we group methods together in functional groups?  These methods up here are handle operations.  The group below are id operations.  Also rename containerID to id.  We should try not to refer to them as containers in this new proxy since they apply to both containers and pods.